src/interface.js

```diff
@@ -0,0 +1,93 @@
+$('document').ready(function runDocument() {
+  let scorecard = new Scorecard(Frame, FinalFrame);
+
+  refreshPoints = function refreshPoints() {
+    for (let i = 0; i < 10; i += 1) {
+      const frame = scorecard.frames[i];
+      for (let j = 0; j < frame.rolls.length; j += 1) {
+        if (frame.rolls[j] === 10) {
+          $('#frame' + i + 'roll' + j).text('X');
```

Comment:
See comment near the top about making presentation code DRY - if you use view classes then a lot of this becomes simpler and you have fewer nested if statements.